### DISCUSSION TITLE
uBlock Origin german addon description should be updated

### DISCUSSION THREAD

**Discussion by JobcenterTycoon** [uBlockOrigin/uBlock-issues#2921] [General]
Hello.

While the EN description on https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/ is fine and up to date the german description https://addons.mozilla.org/de/firefox/addon/ublock-origin/ is completely outdated:

> Standardmäßig werden folgende Filterlisten geladen und angewandt:
> - EasyList
> - Peter Lowe’s Ad server list
> - EasyPrivacy
> - Malware domains
> 
> Auf Wunsch können zusätzliche Listen ausgewählt werden:
> 
> - Fanboy’s Enhanced Tracking List
> - Dan Pollock’s hosts file
> - hpHosts’s Ad and tracking servers
> - MVPS HOSTS
> - Spam404
> - etc.

The list should be updated. This sentence also seems to be obsolete:

> Dies ist eine ziemlich frühe Version - bitte denke daran, wenn du sie bewertest.

**Comment by gorhill** [comment]
I don't have an updated version at Crowdin, can you provide me one here?

**Reply by JobcenterTycoon** [reply]
Change

```
Standardmäßig werden folgende Filterlisten geladen und angewandt:

- EasyList
- Peter Lowe’s Ad server list
- EasyPrivacy
- Malware domains

Auf Wunsch können zusätzliche Listen ausgewählt werden:

- Fanboy’s Enhanced Tracking List
- Dan Pollock’s hosts file
- hpHosts’s Ad and tracking servers
- MVPS HOSTS
- Spam404
- etc.
```

to

```
Standardmäßig werden folgende Filterlisten geladen und angewandt:

- EasyList (Werbung)
- EasyPrivacy (Tracking)
- Peter Lowe’s Ad server list (Werbung und Tracking)
- Online Malicious URL Blocklist
- uBO's eigene Filterlisten

Auf Wunsch können zusätzliche Listen ausgewählt werden:

- EasyList Cookie
- Fanboy Annoyances
- AdGuard Annoyances
- Dan Pollock’s hosts file
- Und viele weitere
```

And delete

```
Dies ist eine ziemlich frühe Version - bitte denke daran, wenn du sie bewertest.
```

**Reply by gorhill** [reply]
Alright, updated, I also removed link to very dated wiki page, and two other paragraphes for being too verbose.

**Reply by JobcenterTycoon** [reply]
The description for Chrome, Edge/Microsoft and Opera also need to be updated. But as far i see its exacly the same.

**Reply by gorhill** [reply]
For CWS, I will wait to do this at the same time I publish 1.53.0, because just updating the description causes a pending review period preventing me from doing anything until it passes review.
